Assets/Plugins/Android/Editor/PlatformSpecificConfigEditorAndroid.cs

```diff
@@ -28,34 +28,11 @@
 namespace PlayEveryWare.EpicOnlineServices
 {
 
-    public class PlatformSpecificConfigEditorAndroid : IPlatformSpecificConfigEditor
+    public class ConfigSectionAndroid : ConfigSection<EOSAndroidConfig>
```

Comment:
This is an artifact of renaming the interface `IPlatformSpecificConfigEditor` to ultimately be an abstract class `ConfigSection` because ReSharper wanted to be helpful and rename more than I asked it to. Consequently, the filename no longer matches the class name - so this should be fixed and is reason enough to request changes.